include/swift/AST/DiagnosticsSema.def

```diff
@@ -1343,6 +1344,10 @@ WARNING(pattern_type_not_usable_from_inline_warn,none,
         "%select{%select{variable|constant}0|property}1 "
         "should be '@usableFromInline' or public",
         (bool, bool))
+ERROR(pattern_type_not_usable_from_inline_fixed_layout,none,
+      "type referenced from a stored property in a @_fixedLayout struct must "
```

Comment:
Nitpick: `@_fixed_layout`

Reply:
Grr. That's what I get for using the new term in comments and the original spelling in attributes.